### ISSUE TITLE
ConfigurationTest fails when installing via composer

### ISSUE THREAD

**Issue by barbazul** [magento/magento2#12574]
An integration test is failing when Magento is built via composer (not when clones from github).

The problem seems to be that basePath is calculated as the app directory but the actual code is in vendor so XML configuration is not found and an uncatched exception is thrown at https://github.com/magento/magento2/blob/276c690d6ac6e97d58fe305aefb071a7b386be2a/dev/tests/integration/testsuite/Magento/Ui/Component/ConfigurationTest.php#L87

### Preconditions
1.  Build magento via command line as instructed in the devdocs
2. Set up integration tests as instructed in the devdocs

### Steps to reproduce

1.  `php vendor/bin/phpunit -c dev/tests/integration --filter ConfigurationTest`

### Expected result

1.  Test passes

### Actual result

```
PHPUnit 6.2.4 by Sebastian Bergmann and contributors.

E                                                                   1 / 1 (100%)

Time: 5.17 seconds, Memory: 218.59MB

There was 1 error:

1) Magento\Ui\Component\ConfigurationTest::testConfiguration
Magento\Framework\Exception\FileSystemException: Cannot read contents from file "/var/www/html/app/var/www/html/vendor/magento/module-email/view/adminhtml/ui_component/design_config_form.xml"

/var/www/html/vendor/magento/framework/Filesystem/Driver/File.php:149
/var/www/html/vendor/magento/framework/Filesystem/Directory/Read.php:214
/var/www/html/dev/tests/integration/testsuite/Magento/Ui/Component/ConfigurationTest.php:87
/var/www/html/vendor/phpunit/phpunit/phpunit:53

ERRORS!
Tests: 1, Assertions: 0, Errors: 1.

=== Memory Usage System Stats ===
Memory usage (OS):      245.58M (113.69% of 216.00M reported by PHP)
Estimated memory leak:  29.58M (12.04% of used memory)
```

**Comment by magento-engcom-team** [comment]
@barbazul, thank you for your report.
We've created internal ticket(s) MAGETWO-85719 to track progress on the issue.

**Comment by okorshenko** [comment]
Hi @barbazul. Thank you for your report. 
The issue has been fixed in magento-engcom/magento2ce#1161 by @nmalevanec in 2.2-develop branch
Related commit(s):
 - 29e3f33f34000c0f69268e19913a76488a75e432

The fix will be available with the upcoming 2.2.4 release.
